sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/OpenTelemetryConfigurationFactory.java

```diff
@@ -31,9 +31,9 @@ static OpenTelemetryConfigurationFactory getInstance() {
   public OpenTelemetrySdk create(
       OpenTelemetryConfigurationModel model, SpiHelper spiHelper, List<Closeable> closeables) {
     OpenTelemetrySdkBuilder builder = OpenTelemetrySdk.builder();
-    if (!"0.3".equals(model.getFileFormat())) {
+    if (!"0.4".equals(model.getFileFormat())) {
```

Comment:
Is `"0.4" constant-worthy?

Reply:
I don't have a strong feeling either way. Currently its only referenced in this method and in test code, and I think it makes sense for all the test code references to explicitly use `"0.4"` rather than a constant.

Still, a constant makes it very clear that this is an important value.